bootstrap/090_post_setup.sql

```diff
@@ -387,6 +384,9 @@ DECLARE
 BEGIN
     let query_id text := (select query_id from table(information_schema.task_history(TASK_NAME => :task_name, ROOT_TASK_ID => :root_task_id)) WHERE GRAPH_RUN_GROUP_ID = :task_run_id  AND DATABASE_NAME = current_database() limit 1);
 
+    -- Refresh the warehouses prior to refreshing the warehouse load history
+    call internal.refresh_warehouses();
```

Comment:
if we're calling here, can we remove any other calls/tasks that do this?

Reply:
> if we're calling here, can we remove any other calls/tasks that do this?

I think it was just the one I removed in this patch, but i'll dbl check.

Reply:
wait, why don't you move the operation into warehouse_load_history? Isn't that where we need it?

Reply:
Sorry, maybe I'm missing something: that's what I did here. I moved it from SIMPLE_DATA_EVENTS_MAINTENANCE to WAREHOUSE_LOAD_MAINTENANCE. We will refresh our table of warehouses and then we refresh the WLH for each wh in that table.

Reply:
ah... misleading github snippet. makes sense.